frontend/src/layouts/ProtectedLayout.vue

```diff
@@ -6,16 +6,30 @@ import SideNav, {
     type ISideNavItem,
 } from '@/components/common/SideNav.vue';
 import sideNavData from '@/static/sideNav.json';
-import { isApplicationSelected } from '@/store/ApplicationState';
+import { isApplicationSelected,selectedApplicationShortDisplayText } from '@/store/ApplicationState';
 
 const navigationData = ref<[ISideNavData]>(sideNavData as any);
 
+const initialLabel = () => {
+    if(selectedApplicationShortDisplayText.value === 'FAM') {
```

Comment:
Perhaps this application 'FAM' now should be a constant in frontend to be used on other places.

Reply:
Catherine suggested that, already put it into a costant